test/Interop/Cxx/templates/class-template-instantiation-in-swift.swift

```diff
@@ -0,0 +1,16 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop)
+//
+// REQUIRES: executable_test
+
+import UsingDirective
+import StdlibUnittest
+
+var TemplatesTestSuite = TestSuite("TemplatesTestSuite")
+
+TemplatesTestSuite.test("without-definition") {
+  var intWrapper = IntWrapper(value: 42)
+  var wrappedMagicNumber = MagicWrapper<IntWrapper>(t: intWrapper)
+  expectEqual(wrappedMagicNumber.getValuePlusArg(8), 50)
+}
```

Comment:
Would you mind adding another test for `MagicWrapper<Int>` (or some test where T = a swift type)?

Reply:
Done. I added a test using CInt as a param. And in error reporting test we have a test showing the error message when importing a Swift type that cannot be (currently) converted - Optional. 

I just discovered that this PR fixes https://bugs.swift.org/browse/SR-13775, so I uncommented the test and removed the early bail-out added by https://github.com/apple/swift/pull/34428. \o/

Reply:
Great! Thanks.